python/ray/data/_internal/plan.py

```diff
@@ -270,11 +271,11 @@ def get_plan_as_string(self) -> str:
                         )
                     schema_str.append(f"{col_str}")
                 schema_str = ",\n".join(schema_str)
-                schema_str = "{\n" + schema_str + "\n\t}"
+                schema_str = "{\n" + schema_str + f"\n{INDENT_STR}}}"
             dataset_str = (
-                "Dataset(\n\tnum_blocks={},\n\tnum_rows={},\n\tschema={}\n)".format(
-                    num_blocks, count, schema_str
-                )
+                f"Dataset(\n{INDENT_STR}num_blocks={num_blocks},"
```

Comment:
Can we follow the other logic in this method, to take `trailing_space` into consideration? We should also add trailing spaces as there might be more operators.

example:

```
ds = ray.data.read_csv("/Users/chengsu/ray/python/ray/data/examples/data/iris.csv")
ds = ds.map_batches(lambda x:x)
ds = ds.map_batches(lambda x:x)
ds = ds.map_batches(lambda x:x)
ds = ds.map_batches(lambda x:x)
ds = ds.map_batches(lambda x:x)
ds

MapBatches(<lambda>)
+- MapBatches(<lambda>)
   +- MapBatches(<lambda>)
      +- MapBatches(<lambda>)
         +- MapBatches(<lambda>)
            +- Dataset(
    num_blocks=1,
    num_rows=150,
    schema={
        sepal.length: double,
        sepal.width: double,
        petal.length: double,
        petal.width: double,
        variety: string
    }
)

```